index.html

```diff
@@ -17,202 +18,213 @@
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Join</a>
+              </li>
+
+              <li class="nav-item">
+                <a class="nav-link" href="#">Sponsor</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">News</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">CC Campaign</a>
+              </li>
+            </ul>
+            <form class="d-flex mt-3" role="search">
+              <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search">
+              <button class="btn btn-outline-success" type="submit">Search</button>
+            </form>
+          </div>
         </div>
       </div>
+    </nav>
+
+  </div>
+
+  <nav class="nav-desktop2 ">
+
+    <div class="nav-desktop2-margin">
+      <a href="https://www.facebook.com"><i class="bi bi-facebook"></i></a>
+      <a href="https://twitter.com/?lang=es"><i class="bi bi-twitter"></i></a>
+      <a href="about.html">Logout</a>
+      <a href="#about">Mypage</a>
+      <a href="#contact">English</a>
     </div>
+
+    
+
+
+
+  </nav>
+  <nav class="nav-desktop ">
+    <h5 class="offcanvas-title" id="offcanvasNavbarLabel"><img src="./images/Logo.png" alt=""></h5>
+    <a href="about.html">About </a>
+    <a href="#about">Program</a>
+    <a href="#contact">Join</a>
+    <a href="#portfolio">Sponsor</a>
+    <a href="#portfolio">News</a>
+    <a href="#portfolio">CC Campaign</a>
+
+
   </nav>
   <section class="background-intro">
-    <div class="container-md cocogoose">
-      <h2 class="h2-intro">"Hello!! Sharing world of Knowledge!!"</h2>
-      <h1 class="h1-intro">Web Programming course's for everyone 2023</h1>
-    </div>
-    <div class="container containerpro-1">
-      <p>
-        A joyful program that believe in the peer to peer programming to learn, creating a positive work with people all
-        around the world.
-      </p>
+    <div class="container cocogoose">
+
+      <div class="margin-text1">
+        <h2 class="h2-intro">"Hello!! Sharing world of Knowledge!!"</h2>
+        <h1 class="h1-intro" id="title">WEB PROGRAMMING COURSE'S FOR EVERYONE 2023</h1>
+      </div>
+      
     </div>
-    <div class="container-md">
-      <p class="p-intro">2023.03.17(MON) ~ 18(TUE)</p>
-      <p class="p-intro2"> @National Museum of Bogotá</p>
+<div class="margin-text2">
+  <div class="containe containerpro-1">
+    <p>
+      A joyful program that believe in the peer to peer programming to learn, creating a positive work with people all
+      around the world.
+    </p>
+  </div>
+</div>
+    
+    <div class="margin-text2">
+      <div class="container">
+        <p class="p-intro"><strong>2023.03.17(MON) ~ 18(TUE)</strong>  </p>
+        <p class="p-intro2"> <strong> @ Neiva-Huila-Colombia</strong> </p>
+      </div>
+
     </div>
+    
   </section>
```

Comment:
- [ ] I can see you added some of the required changes but there is something you are till missing out. As requested by the previous reviewer kindly consider making the background image of this section transparent to implement exactly  the provided design check the images below for more clarification 
- your design 👇 
<img width="1412" alt="Screen Shot 2023-04-20 at 19 57 32" src="https://user-images.githubusercontent.com/60146030/233449523-9c972c99-7127-444e-9026-79d1b26a4ec6.png">


- Provided design 👇 
<img width="967" alt="Screen Shot 2023-04-20 at 19 59 07" src="https://user-images.githubusercontent.com/60146030/233449715-be1acbd9-3907-4a98-bd41-7d3ea8361522.png">

Reply:
Hi, I'm trying ti understand what you are requesting, but I cant, I tried to change the background color to get similar to the design, but honestly, I don't get it.
![image](https://user-images.githubusercontent.com/111823606/233481554-e64ce9f7-9226-4a8b-b4fb-4588b9c8f46f.png)
My background a guess is transparent for the other divs, the only one that I'm seeing that is not is the box where's the paragraph " A joyful celebration believing......", in the design, it has a background color but is not transparent.

![image](https://user-images.githubusercontent.com/111823606/233482096-2daab03a-6a1e-4637-b484-c2816ecaee75.png)

so, if I put that transparent, it would seem like this:
![image](https://user-images.githubusercontent.com/111823606/233482540-63ddf17c-4d5d-46ff-94ed-4d1974bd5823.png)

So, I'm sorry for my ignorance, but I don't understand this part, can you help me?